docs/samples/custom/torchserve/docs/autoscaling.md

```diff
@@ -0,0 +1,88 @@
+# Autoscaling
+
+## Deployment yaml
+
+For example, specify a “concurrency target” of “5”, the autoscaler will try to make sure that every replica receives on average 5 requests at a time.
+By default the pod scale with concurrency metrics
+
+```yaml
+apiVersion: "serving.kubeflow.org/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: torchserve-custom-autoscaling
+  annotations:
+    autoscaling.knative.dev/target: "5"
+spec:
+  predictor:
+    containers:
+    - image: {username}/torchserve:latest
+      name: torchserve-container
+      ports:
+        - containerPort: 8080
+```
+
+## Create the InferenceService
+
+Apply the CRD
+
+```bash
+kubectl apply -f autoscale.yaml
+```
+
+Expected Output
+
+```bash
+$inferenceservice.serving.kubeflow.org/torchserve-custom-autoscaling created
+```
+
+## Run a prediction
+
+The first step is to [determine the ingress IP and ports](../../../README.md#determine-the-ingress-ip-and-ports) and set `INGRESS_HOST` and `INGRESS_PORT`
+
+```bash
+MODEL_NAME=torchserve-custom-autoscaling
+SERVICE_HOSTNAME=$(kubectl get inferenceservice ${MODEL_NAME} -o jsonpath='{.status.url}' | cut -d "/" -f 3)
+curl -v -H "Host: ${SERVICE_HOSTNAME}" http://${INGRESS_HOST}:${INGRESS_PORT}/predictions/mnist -T 1.png
```

Comment:
I think sending one request will not result in autoscaling, you would need to use `hey` or `vegeta` for the scaling example

Reply:
here is an example https://github.com/kubeflow/kfserving/tree/master/docs/samples/autoscaling#load-inferenceservice-with-concurrent-requests